### PR TITLE
Answer to exercise 1-4 ch.5

### PR DESCRIPTION
**Pull request by ryblovAV**
Hi @axel22 
Pls see my answers

### CODE REVIEW

File: src/main/scala/org/learningconcurrency/exercises/ch5/ex1.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ */
+
+object Ex1 extends App {
+
+  object Timed {
+
+    @volatile 
+    var dummy: Any = _
+
+    def buildObjects(count:Int) = {
```
Comment (axel22) [inline src/main/scala/org/learningconcurrency/exercises/ch5/ex1.scala]: Nit: you could use ScalaMeter here.

**Comment by axel22** [comment]
LGTM

The Cellular Automaton example could be further optimized, but it looks correct to me!

**Comment by axel22** [comment]
Hm, it looks like this broke the build, due to some issues with JavaFX.

https://travis-ci.org/concurrent-programming-in-scala/learning-examples/builds/83789552

**Comment by ryblovAV** [comment]
Hi @axel22,
I added sbt dependency for JavaFX 8, but travis.yml uses oraclejdk7. 
Could you use JDK 8 ?

**Comment by axel22** [comment]
Scala 2.11 provides only experimental support for JDK 8, and is mainly targeting JDK 7. I feel a bit anxious about changing that.

**Pull request merged**